workshops/urls.py

```diff
@@ -20,7 +20,12 @@
         url(r'^membership/add/$', views.membership_create, name='membership_add'),
     ])),
 
+    url(r'^memberships/', include([
+        url(r'^$', views.all_memberships, name='all_memberships'),
+        url(r'^add/$', views.MembershipCreate.as_view(), name='membership_add'),
+    ])),
```

Comment:
We can do away with the other `membership_add` urlpattern by extending the existing `membership_create` view.